SNR/minimal_restructuring.py

```diff
@@ -0,0 +1,89 @@
+# Copyright 2021 SuperDARN Canada, University of Saskatchewan
+# Author: Remington Rohel
+"""
+This file contains functions for "restructuring" site to array Borealis HDF5 files on memory-limited systems,
+for the express purpose of plotting with hdf5_rtplot_utils.py.
+
+Functions
+---------
+"""
+import copy
+import numpy as np
+import os
+import deepdish as dd
+
+
+def antennas_iq_site_to_array(antennas_iq_file, antenna_nums=None):
+    """
+    Gets the minimal fields necessary to build up an array-structured representation of antennas_iq_file
+    for plotting.
+
+    Parameters
+    ----------
+    antennas_iq_file:   str
+        Path to an antennas_iq site-structured file.
+    antenna_nums:       list
+        List of antenna indices to retrieve. Entries should be ints.
+
+    Returns
+    -------
+    arrays: dict
+        Dictionary of necessary fields for plotting.
+    antenna_names: list
+        List of all the antenna names.
+    antenna_indices: list
+        List of all the indices in the data corresponding to antenna_indices.
+    """
+    # Dictionary of fields that will be returned
+    arrays = {}
+
+    group = dd.io.load(antennas_iq_file)
+    timestamps = sorted(list(group.keys()))
+
```

Comment:
```suggestion
ant_key = 'antenna_arrays_order'
```
Just because this string is repeated 5 times in this function
'data_dimensions' also appears a few times